cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMaker2AssemblyOperator.java

```diff
@@ -465,7 +465,7 @@ private static void prepareMirrorMaker2ConnectorConfig(Reconciliation reconcilia
         config.putAll(mirror.getAdditionalProperties());
     }
 
-    private static void addClusterToMirrorMaker2ConnectorConfig(Map<String, Object> config, KafkaMirrorMaker2ClusterSpec cluster, String configPrefix) {
+    static void addClusterToMirrorMaker2ConnectorConfig(Map<String, Object> config, KafkaMirrorMaker2ClusterSpec cluster, String configPrefix) {
```

Comment:
```suggestion
    /* test */ static void addClusterToMirrorMaker2ConnectorConfig(Map<String, Object> config, KafkaMirrorMaker2ClusterSpec cluster, String configPrefix) {
```

I see it was changed from `private` to `package` (implicit) to be used in tests. In this case we have the convention to add a comment as I suggested here, in order to make clear why it's not `private`.

Reply:
I resolved this in the latest commit